clickhouse_options.go

```diff
@@ -155,11 +156,9 @@ func (o *Options) fromDSN(in string) error {
 			InsecureSkipVerify: skipVerify,
 		}
 	}
+	o.Scheme = dsn.Scheme
 	switch dsn.Scheme {
 	case "http":
-		if secure {
```

Comment:
still wonder if we should error here..user specifies secure it should be SSL

Reply:
Scheme will be changed to HTTPS

Reply:
This is going to lead to confusing behaviour...a user will have specified http and a http port. We switch the scheme to https due to the secure parameter and we will later error. I'd prefer we just return an error and state "secure must be performed over https".